### PR TITLE
GreenIO Blog Post

### PR DESCRIPTION
**Pull request by dkrees**
Please add a direct link to your post here:

https://dkrees.github.io/SL-blog/2024/10/04/greenio-conference.html

Have you (please tick each box to show completion):

 - [x] Added your blog post to a single category?
 - [x] Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
 - [x] Checked that the build passes?
 - [x] Checked your spelling (you can use `npm install` followed by `npx mdspell "**/{FILE_NAME}.md" --en-gb -a -n -x -t` if that's your thing)
 - [x] Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
 - [x] Optimised any images in your post? They should be less than 100KBytes as a general guide.

Posts are reviewed / approved by your Regional Tech Lead.

### CODE REVIEW

File: _posts/2024-10-04-greenio-conference.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Green IO London 2024
+
+Oliver Cronk and I recently attended the Green IO London event, where we joined a passionate group of like-minded individuals to explore and discuss sustainable technology.
+
+The event featured a full day of talks by a diverse range of speakers, covering a wide variety of subjects. The full line up of speakers and their slides are available on [GreenIO](https://greenio.tech//conference/9/london-2024-september-green-it)
+
+## Key Takeaways
+
+A major theme of the event was the urgent need for standardised and consistent data to measure emissions more accurately. The consensus is that without reliable data, organisations struggle to make informed decisions about third-party services and internal processes that could mitigate their digital carbon footprint.
+There were also discussions on the misleading nature of greenwashing, where companies report market-based carbon emissions that don't fully reflect their actual environmental impact. This highlights the need for more transparency and accurate reporting to truly reduce digital emissions.
```
Comment (cjohns-scottlogic) [inline _posts/2024-10-04-greenio-conference.md:25]: This a newline in the .md but the resulting webpage has it all as one block. Was the intention to have a blank line here?

File: _posts/2024-08-06-carbon-emissions-of-end-user-devices-part-two.md
Comment (cjohns-scottlogic) [inline _posts/2024-08-06-carbon-emissions-of-end-user-devices-part-two.md:1]: Was the file deletion intentional?

Comment (dkrees) [reply _posts/2024-08-06-carbon-emissions-of-end-user-devices-part-two.md:1]: er... no? No idea why that happened. Thanks for spotting that!

File: _posts/2024-10-04-greenio-conference.md (hunk 1 above)
Comment (dkrees) [reply _posts/2024-10-04-greenio-conference.md:25]: Yes, that should have been separated. Copy & Paste hangover.

**Review by cjohns-scottlogic — APPROVED** [review]

**Review by cjohns-scottlogic — APPROVED** [review]

**Pull request merged**